resources/views/people/_header.blade.php

```diff
@@ -93,6 +93,9 @@
             <li>
               <a href="{{ route('people.edit', $contact) }}" class="btn edit-information" id="button-edit-contact">{{ trans('people.edit_contact_information') }}</a>
             </li>
+            <li>
+              <contact-archive hash="{{ $contact->hashID() }}" active="{{ json_encode($contact->is_active) }}"></contact-archive>
```

Comment:
I don't think it should be put this feature here in the header. This is not a common action or even an important one. I would put it in the sidebar, just above the `Delete contact` option. You don't want people to easily access this feature, in my opinion.